crates/program/src/error.rs

```diff
@@ -0,0 +1,28 @@
+use thiserror::Error;
+
+use std::fmt;
+
+/// Represents error that may occur while doing gas estimation
+#[derive(Debug, PartialEq, Clone, Error)]
+pub enum ProgramError {
+    /// Invalid wasm
+    InvalidWasm,
+    /// No valid `svm_alloc` function found.
+    FunctionNotFound { func_name: String },
+    /// Floats not allowed
+    FloatsNotAllowed,
+    /// Too many function imports
+    TooManyFunctionImports,
+    /// Function index is too large
+    FunctionIndexTooLarge,
+    /// `call_indirect` isn't allowed
+    CallIndirectNotAllowed,
```

Comment:
This is part of the `Fixed Gas` (disallowing the `call_indirect` opcode)

Reply:
Ops, missed it!